modules/live-schedule-impl/src/main/java/org/opencastproject/liveschedule/impl/LiveScheduleServiceImpl.java

```diff
@@ -450,29 +450,43 @@ void publishToSearch(MediaPackage mp) throws LiveScheduleException {
   }
 
   void retract(MediaPackage mp) throws LiveScheduleException {
+    Organization org = securityService.getOrganization();
+    User prevUser = org != null ? securityService.getUser() : null;
     try {
-      List<Job> jobs = new ArrayList<Job>();
+      securityService.setUser(SecurityUtil.createSystemUser(systemUserName, org));
```

Comment:
There is another place in the liveSchedulerServiceImpl where this approach is used. So I used it, too.